packages/firestore/src/local/memory_mutation_queue.ts

```diff
@@ -173,6 +174,16 @@ export class MemoryMutationQueue implements MutationQueue {
     return PersistencePromise.resolve(this.findMutationBatch(batchId));
   }
 
+  lookupMutationKeys(
+    transaction: PersistenceTransaction,
+    batchId: BatchId
+  ): PersistencePromise<DocumentKeySet | null> {
+    const mutationBatch = this.findMutationBatch(batchId);
```

Comment:
Should we assert that the mutationBatch is found since the memory mutation queue is only used in non-multi-tab and so it would be a bug to look up a mutation's keys after it's removed?

Reply:
Yes, that makes sense. Albeit, this method should never be called in the first place.

I added a check for the tombstone-status instead.